### ISSUE TITLE
DotNetCoreTest overwrites prior defined arguments

### ISSUE THREAD

**Issue by cmenzi** [Romanx/Cake.Coverlet#2]
When using the `DotNetCoreTest` in conjunction with `CoverletSettings` the previous defined arguments are thrown away.

```
    var settings = new DotNetCoreTestSettings {
        Configuration = configuration,
        OutputDirectory = outputDir,
        NoBuild = true,
        ArgumentCustomization = args => args.Append("-l trx")
                                            .Append($"-r {testResultsDir.ToString()}")
    };
    var coverletSettings = new CoverletSettings {
        CollectCoverage = !skipCoverage,
        CoverletOutputFormat = CoverletOutputFormat.opencover,
        CoverletOutputDirectory = testResultsDir,
        CoverletOutputName = $"coverage"
    };
    DotNetCoreTest(project.FullPath, settings, coverletSettings);
```
The args `-l trx -r testresults` gets no appended anymore.

I think the bad line is https://github.com/Romanx/Cake.Coverlet/blob/master/src/Cake.Coverlet/CoverletAliases.cs#L34 where you overwrite the delegate.

**Comment by Romanx** [comment]
Hi there,

I thought that might happen. I'll fix it and get a release out tonight. Thanks for reporting it!

**Comment by Romanx** [comment]
@cmenzi I've made the fix and it's currently in the 1.1 feature branch. It's currently held up by #1 but as soon as I decide how to handle that edge case i'll release a fix. Thanks

**Comment by Romanx** [comment]
@cmenzi this has just been released to nuget. Pleae let me know if you have any issues.
